py-rse/tree.Rmd

```diff
@@ -0,0 +1,144 @@
+# Project Tree {#tree}
+
+The final directory tree for the Zipf's Law project looks as follows:
+
+```
+zipf/
+├── .gitignore
+├── CITATION.md
+├── CONDUCT.md
+├── CONTRIBUTING.md
+├── KhanVirtanen2020.md
+├── LICENSE.md
+├── Makefile
+├── README.rst
+├── environment.yml
+├── requirements.txt
+├── requirements_docs.txt
+├── setup.py
+├── data
+│   ├── README.md
+│   ├── dracula.txt
+│   ├── frankenstein.txt
+│   ├── jane_eyre.txt
+│   ├── moby_dick.txt
+│   ├── sense_and_sensibility.txt
+│   ├── sherlock_holmes.txt
+│   └── time_machine.txt
+├── docs
+│   ├── Makefile
+│   ├── conf.py
+│   ├── index.rst
+│   └── source
+│   │   ├── collate.rst
+│   │   ├── countwords.rst
+│   │   ├── modules.rst
+│   │   ├── plotcounts.rst
+│   │   ├── test_zipfs.rst
+│   │   └── utilities.rst
+├── results
+│   ├── collated.csv
+│   ├── collated.png
+│   ├── dracula.csv
+│   ├── dracula.png 
+│   ├── frankenstein.csv
+│   ├── jane_eyre.csv
+│   ├── jane_eyre.png 
+│   ├── moby_dick.csv
+│   ├── sense_and_sensibility.csv
+│   ├── sherlock_holmes.csv
+│   └── time_machine.csv
+├── test_data
+│   ├── random_words.txt
+│   └── risk.txt
+└── zipf
+    ├── book_summary.sh
+    ├── collate.py
+    ├── countwords.py
+    ├── plotcounts.py
+    ├── plotparams.yml
+    ├── test_zipfs.py
+    └── utilities.py
+```
+
+Each file was introduced and subsequently modified
+in the following chapters, sections and exercises:
+
+`.gitignore`: Introduced in Section \@ref(git-cmdline-ignore).
```

Comment:
Would this be better as a list? Or as a table?

Reply:
It would possibly be better as a table, but I figured I'd leave it like this at the moment and the book editors can indicate how best to present the information.